tweetprocessors.js

```diff
@@ -113,13 +115,86 @@ class HashtagStats {
 
   /**
    * Returns a list of all the trending hashtags and countries.
+   * The list is ordered by country, hashtags with highest count.
+   * It is formatted as a list of [country, hashtag, entry].
+   */
+  get() {
+    return new Promise((resolve, reject) => {
+      this.HashtagStats.findAll({order: ['country', 'count']}).then(full_list => {
+        resolve(full_list.map(elem => [elem.dataValues.country, elem.dataValues.hashtag, elem.dataValues.count]).reverse());
+      });
+    });
```

Comment:
When a value is returned within a `then` call, it will automatically be wrapped in a promise. Thus, we can simply write this as:
```js
return this.HashtagStats.findAll({order: ['country', 'count']}).then(full_list => full_list.map(elem => [elem.dataValues.country, elem.dataValues.hashtag, elem.dataValues.count]).reverse());
```
(Broken up into multiple lines, of course.)